spec/factories/miq_template.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
   factory :miq_template do
+    name "ubuntu-16.04-stable"
+    location "Minneapolis, MN"
+    vendor "openstack"
```

Comment:
It is nice to keep templates at the bare minimum.

Could we create a separate template for a `template_with_location` or something?
or maybe an `openstack_miq_template`?